packages/ripple-keypairs/src/types.ts

```diff
@@ -0,0 +1,37 @@
+export type HexString = string
+export type Algorithm = 'ecdsa-secp256k1' | 'ed25519'
+export type KeyType = 'public' | 'private'
+
+export interface KeyPair {
+  privateKey: HexString
+  publicKey: HexString
+}
+
+export interface DeriveKeyPairOptions {
+  validator?: boolean
+  accountIndex?: number
+}
+
+export interface SigningScheme {
+  deriveKeypair: (
+    entropy: Uint8Array,
+    options?: DeriveKeyPairOptions,
+  ) => KeyPair
+
+  sign: (
+    // TODO: HexString?
```

Comment:
Flagging HexString TODO (similar one below in `verify`)

Reply:
https://github.com/XRPLF/xrpl.js/pull/2273/commits/6f5ddb566d3fd48d2f73c609e62ffe90653db06a

Reply:
Reverted to address in another PR

Reply:
should this TODO be removed then?